EvotoApi/App_Start/Startup.Auth.cs

```diff
@@ -27,6 +27,10 @@ public void ConfigureAuth(IAppBuilder app)
             {
                 AuthenticationType = DefaultAuthenticationTypes.ApplicationCookie,
                 LoginPath = new PathString("/manage/login"),
+                SlidingExpiration = true,
+                CookieHttpOnly = true,
```

Comment:
Cookie HTTP means it'll only get the cookies from the request from the client, it won't handle any cookies attached separately. So HTTP/HTTPS both come under this so it's fine.